packages/web3-eth/test/integration/defaults.transactionBlockTimeout.test.ts

```diff
@@ -102,43 +94,31 @@ describe('defaults', () => {
 				// eslint-disable-next-line jest/no-conditional-expect
 				expect((error as Error).message).toMatch(/was not mined within [0-9]+ blocks/);
 			}
```

Comment:
These `try/catch`es should be refactored to look like [this](https://github.com/web3/web3.js/blob/82e9b03ca7396b63863761678c93dbf4dc0ed8a7/packages/web3-eth/test/integration/web3_eth/send_transaction.test.ts#L544-L548) so the test will fail on it's own if no error is received - also these checks should be testing the `error` event is emitted as expected

But this isn't directly related to the purpose of this PR, so an issue could be made to follow up on these changes

Reply:
done